### ISSUE TITLE
Search inaide file and hightlight all found entries

### ISSUE THREAD

**Issue by Kimi-Arthur** [k0shk0sh/FastHub#967]
**FastHub Version: 4.2.0**  
**Android Version: 7.0 (SDK: 24)**  
**Device Information:**  
- **Manufacturer:** Xiaomi  
- **Brand:** Xiaomi  
- **Model:** MI MAX  
---

The search function works fine to get to the file. But since I cannot find the line and all appearences. It will be great if it can.

**Comment by yakov116** [comment]
@Kimi-Arthur 
> and all appearances
What do you mean by that?

From what I am understanding you are asking for a goto line option?

Thanks for your feedback

**Comment by Kimi-Arthur** [comment]
No, it's still search. Like when you search in Chrome, all occurrences are highlighted.

Sorry if my description isn't clear enough.

**Comment by k0shk0sh** [comment]
he meant search in files 

🤷‍♂️

**Comment by avently** [comment]
waiting this feature for a couple of months. I have to use web version of GitHub for the search inside file.

**Comment by markddavidoff** [comment]
+1
